pages/index.vue

```diff
@@ -41,8 +35,12 @@ export default {
 			]
 		};
 	},
-	fetch ({ store }) {
-		return store.dispatch("fetchSitePages");
+	asyncData ({ $axios, $payloadURL, route, store }) {
+		if (process.static && process.client && $payloadURL) {
+			return $axios.$get($payloadURL(route));
+		}
+
+		return store.state.sitePages.find(oneSitePage => oneSitePage.slug === "home");
```

Comment:
Same as above: add `await store.dispatch("fetchSitePages");` before this line?

Reply:
No need because it is called in `actions.js` within `nuxtServerInit`

Reply:
Makes sense. Thanks.